examples/erc-7579/lib/permissionless.ts

```diff
@@ -16,8 +16,9 @@ import {
   createPimlicoPaymasterClient
 } from 'permissionless/clients/pimlico'
 import { EntryPoint } from 'permissionless/types'
+import { rpcUrl } from './safe'
 
-export type SafeSmartAccountClient = SmartAccountClient<
+export type PermissionlessClient = SmartAccountClient<
```

Comment:
This type is also imported from permissionless, right? I would say, then don't define it in the consumer code.

Reply:
No it's not! Sadly we need to pass many type parameters to the SmartAccountClient type so TS does not complain...